android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1816,28 +1816,36 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         if (KMManager.shouldAllowSetKeyboard()) {
           if (SystemKeyboard.keyboardPickerEnabled) {
             KeyguardManager km = (KeyguardManager) appContext.getSystemService(Context.KEYGUARD_SERVICE);
```

Comment:
```suggestion
            KeyguardManager kgm = (KeyguardManager) appContext.getSystemService(Context.KEYGUARD_SERVICE);
```

`km` is too close to `Keyman` :)

Reply:
I went with a longer `keyguardManager` for disambiguation :)